cmd/ddev/cmd/config.go

```diff
@@ -35,6 +52,12 @@ var ConfigCommand = &cobra.Command{
 		err = c.Write()
 		if err != nil {
 			util.Failed("Could not write ddev config file: %v\n", err)
+
+		}
+
+		// If a provider is specified, prompt about whether to do an import after config.
+		if provider != "" {
+			util.Success("Configuration complete. You may now run `ddev start` or `ddev pull`")
```

Comment:
I'm ok with a followup. Backticks are definitely not the way to communicate with our users. And we absolutely do need punctuation, I think people would expect double or single quotes, which are what was always used pre-markdown (and by people who don't use markdown).

Note that I would assume lots of the things you found in the grep were in our docs where they *do* belong.